packages/terra-notification-dialog/translations/nl-BE.json

```diff
@@ -1,7 +1,7 @@
 {
-  "Terra.notification.dialog.advisory": "Adviserend",
```

Comment:
This string was added when the component was created: https://github.com/cerner/terra-framework/pull/162
I couldn't find any information about why it was added. It is not being used by the component so I removed it